RELEASE_PROCEDURE.md

```diff
@@ -0,0 +1,20 @@
+# Release Procedure
+
+the `jandex` project relies on a GitHub action to create its release.
+
+# Create a PR to prepare the release
+
+By convention, use a branch named `release-<x.y.z>` where `x.y.z` is the version that is about to be released.
+
+The PR should only contain changes to the `.github/project.yml` file and specifies the version to release, and the next version to set in the POM (with a trailing `-SNAPSHOT`).
+
+This PR must be pushed to the [`smallrye/jandex`](https://github.com/smallrye/jandex) repository. Pushing it to a fork will not create a release.
+
+Once this PR is opened, it will check that it can be used to create a release.
+
+Once merged, a GitHub action will update the project and and perform the release to deploy the artifacts in [Maven Central](https://repo1.maven.org/maven2/io/smallrye/jandex/).
+
+
+# Update GitHub release notes
+
+The GitHub action creates a tag corresponding to the release. We can then use GitHub UI to create release notes from this tag.
```

Comment:
The GitHub action actually also creates the release notes, so we don't have to do that manually. Other than that, the description is perfect. Thanks!

Reply:
fixed, thanks